src/app/components/HoldRequest/Notification.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import AppConfigStore from '../../stores/AppConfigStore';
+
+const Notification = () => {
+  const { holdRequestNotification } = AppConfigStore.getState();
+
+  return (
+    <div className="nypl-banner-alert">
+      <p style={{ padding: '10px 20px 0px', margin: 0 }} dangerouslySetInnerHTML={{__html: holdRequestNotification }} />
```

Comment:
`dangerously`? I assume we're not doing anything too crazy here. Does that just allow actual HTML, rather than escaping it?

Reply:
Yes, this allows html. Should be safe as long as we are the only ones supplying the html.